src/main/java/net/datafaker/providers/videogame/RedDeadRedemption2.java

```diff
@@ -0,0 +1,51 @@
+package net.datafaker.providers.videogame;
+
+import net.datafaker.providers.base.AbstractProvider;
+
+/**
+ * Red Dead Redemption 2 is an action-adventure game developed and published by Rockstar Games.
+ *
+ * @since 1.8.0
```

Comment:
Can’t be. 1.8.1 is already released. Upcoming is 2.0.0

Reply:
Ofc! 😅